taxonomies/ingredients.txt

```diff
@@ -268,7 +269,7 @@ cs:Kyselina
 da:Syre
 de:Säuerungsmittel
 el:Μέσο οξίνισης
-es:acidulante, acidulantes
+es:acidulante, acidulantes,acidificante
```

Comment:
Those we are moving to ingredients_classes.txt , can you verify if the entries you added oar already in ingredients_classes.txt and put them there if not?

Reply:
Where is ingredients_classes.txt? It's not in the taxonomies folder..

Reply:
additives_classes.txt is the one you're looking for.

Reply:
Thanks, moved there.